pkg/helper/validate/validate.go

```diff
@@ -56,3 +57,22 @@ func TaskRefExists(spec v1alpha1.TaskRunSpec) string {
 
 	return spec.TaskRef.Name
 }
+
+// Check if step is in waiting, running, or terminated state by checking StepState of the step.
+func StepReasonExists(state v1alpha1.StepState) string {
+
+	if state.Waiting == nil {
+
+		if state.Running != nil {
+			return "Running"
```

Comment:
Open to suggestions is there's a better message that can be returned here. The issue centers around that this will display when the container is not actually executing. `state.Running != nil` when the pod is initialized and container is ready.

Reply:
@danielhelfand yeah for now this is the best we can do I think…